doc/api/configuration.md

```diff
@@ -241,3 +241,33 @@ print(res)
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
 ```
+
+## `persist_snippets`
+
+```{versionadded} 0.8.1
+```
+
+Default: `False`
+
+Stores the snippet in a SQL file.
+
+```{code-cell} ipython3
+%config SqlMagic.persist_snippets = True
+```
+
+```{code-cell} ipython3
+%%sql --save rated_high
+select * from languages where rating > 10.5
+```
+
+The sql file will be saved at: 
+
+`jupysql-snippets/rated_high.sql` 
+
+with the following content: 
+
+`select * from languages where rating > 10.5`
+
```

Comment:
you can run actual code cells to show the content of the file, instead of hardcoding it

Reply:
Making the change